oci/private/structure_test.bzl

```diff
@@ -0,0 +1,62 @@
+"""Loading oci_tarball into the daemon and executes structure_test executable
+
+For some reason if DOCKER_HOST needs to be specified, use --test_env=DOCKER_HOST='unix://<path_to_sock>'
+Beware that this is going to lead to cache misses.
+"""
+attrs = {
+    "image": attr.label(mandatory = True, allow_single_file = True, doc = "Label to an oci_image target"),
+    "config": attr.label_list(allow_files = True, mandatory = True),
+}
+
+CMD = """\
+#!/usr/bin/env bash
+set -o pipefail -o errexit -o nounset
+
+readonly ST_PATH="{st_path}"
+readonly FIXED_ARGS="{fixed_args}"
+
+"$ST_PATH" test $FIXED_ARGS $@
+"""
+
+def _structure_test_impl(ctx):
+    st = ctx.toolchains["@contrib_rules_oci//oci:st_toolchain_type"].st_info
```

Comment:
Same as `default` above. At least use `st_info`. Even though I could argue that `st_info` itself is a bad naming for the provider.